apigentools/commands/generate.py

```diff
@@ -249,4 +261,23 @@ def run(self):
             # after each nested folder has been created
             self.write_dot_apigentools_info(language)
 
-        return 0
+        if self.errors:
```

Comment:
This doesn't feel right to me. If there are any errors while pulling, we probably don't want to skip them and continue regenerating the code. We should make this fail fast and if there are any errors pulling, exit immediately.

Reply:
I did this so that it would generate the other languages/versions if only one failed, but still nicely print all errors at the end.

Reply:
Right now, the way this command works is that it stops at the first error (e.g. if there's an error with the code-generation command itself). I like the fail-fast approach, as it minimizes the risk of potentially doing something wrong to users' local environment when something goes wrong. My opinion is that we should stop doing things as soon as something goes wrong. I'm not sure I have more arguments to support this though... Do you agree or do you feel strongly about making this way the work you want?

Reply:
I don't feel too strongly on this point. My original thought process was to allow the failure of one client to not restrict the generation of the others, but I can understand the want to fail fast in these cases. I updated to reflect that.